docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -79,10 +79,111 @@ on our `feedback form <https://docs.google.com/forms/d/e/1FAIpQLSeCED9TFQsH-1u4Q
 
-.. todo:: 
-
-    In this section we'll discuss the question "What do I do when data sources have different categories?" and how a health 
-    metrics researcher would answer this question and use that answer to inform their methods.
+As noted above, the government provides standardized race and ethnicity 
+categories, though these do change over time. Currently, the race 
+categories are American Indian or Alaska Native, Asian, Black or African American, 
+Native Hawaiian or Other Pacific Islander, and White. The ethnicity 
+categories are Hispanic or Latino and Not Hispanic or Latino. 
+
+By having a set standard, 
+it improves the chance that multiple data sources will have the same 
+categories. However, this is not guarunteed. Here, we will go through 
+a few scenarios of how race and ethnicity categories might be misaligned 
+between datasets and the options for handling each. 
+
+**Nested Categories**
+
+In this guide, we use nested categories to mean that one dataset has more granular 
+categories than another. For example, one dataset might report a race group 
+as simply "Asian" whereas another might have many categories that fit within 
+this group like Chinese, Filipino, Japanese, or Korean. In fact, the US Census 
+often collects both the more granular and rolled up categories of data. 
+
+Granular data is usually better, as different groups within a single race 
+category can have very different experiences and needs. Try to keep the 
+most granular data possible. However, granular data can sometimes create 
+issues with statisitcal power or small sample size. Weigh the different 
+pros and cons as they relate to your project, centering people's 
+identities and needs in the conversation. 
+
+**Non-Nested Categories**
+
+If you need to combine multiple datasets, you will likely have to combine 
+race and ethnicity categories that won't match up perfectly. One option 
+is to take the "least common denominator" approach, which essentially means 
+use the most granular race and ethnicity categories possible that still 
+captures the data fully. 
+
+Often this will end up being the US standard categories above. We strongly 
+recommend against using fewer categories than the US standard unless it 
+is impossible to do otherwise. Creating larger buckets ends up merging 
+individuals with diverse experiences, backgrounds, and identities. Consider 
+what there is to be gained from this analysis and if race and ethnicity are 
+important to include if you plan to use fewer categories than the US standard. 
+
+Another option is to attempt a crosswalking approach if one or more of your 
+input datasets use different categories than the standard. This would allow 
+you to retain all of the data inputs, while keeping some granularity in categories. 
+
+**Multiracial Groups**
+
+Part of the US standard approach is allowing people to select as many race and 
+ethnicity categories as they identify with. Many people have multiracial 
```

Comment:
I'm actually not totally sure whether this is standard or not!

Reply:
I think it is! Here is the link I have been using: https://www.govinfo.gov/content/pkg/FR-1997-10-30/pdf/97-28653.pdf 

And on page 5 they say "When self-identification is used, a method for reporting more than one race should be adopted. The method for respondents to report more than one race should take the form of multiple responses to a single question and not a ‘‘multiracial’’ category."